.travis.yml

```diff
@@ -54,6 +54,10 @@ jobs:
       name: 'Build'
       script:
         - unbuffer node build-system/pr-check/build.js
+    - stage: build
+      name: 'Module Build'
+      script:
+        - unbuffer node build-system/pr-check/module-build.js
```

Comment:
We explicitly don't want this to be in the build phase because a full `dist` plus integration tests take significantly longer than the rest of the tasks in the build phase.

Reply:
OTOH, if the intent is to leave this as just a build without integration tests, ignore all my other comments and carry on 😃

Reply:
Yep, this is just a sanity check that the module build finishes.

Erwin is working on an implementation for more complete integration and other testing.

Reply:
Gotcha. Since the intent is to add tests, let's move this out of the `build` stage and into the `test` stage.

Reply:
Erwin's work will be in the test phase, this work is more similar to the current "build" step... a simple sanity check that the module build can complete.

Reply:
Makes sense. Based on what you say, this job will eventually need to do all the things done by the “Dist, Bundle size” job. Can follow up with a PR that adds the missing stuff.